common/changes/iconame-reducesize_2017-01-10-01-53.json

```diff
@@ -0,0 +1,10 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Icon: -BREAKING CHANGE- Change IconName from an enum to type to greatly reduce file size. Any component that is currently consuming the icon enum will break.",
```

Comment:
Sorry, I meant I'm not sure why we're marking it as "minor" when it's a "major" change. The change itself I agree with completely. We've managed to avoid breaking changings for a long time, but it was bound to happen eventually.